src/languages/bnf.js

```diff
@@ -0,0 +1,29 @@
+/*
+ Language: Backus–Naur Form
+ Author: Oleg Efimov <efimovov@gmail.com>
+ */
+
+function(hljs){
+  return {
+    contains: [
+      // Attribute
+      {
+        className: 'attr',
+        begin: /</,
+        end: />/
+      },
```

Comment:
> How about defining two sides (like in 'ini.js' for example) and then only highlight <..> as attrs on the left and everything else on the right?

Wikipedia highlight <attribute> similar on both sides, I'm just following that style.

> This will be much more robust for detection, too.
It works fine now, so I think we can change this in future if decide.